src/runtime_src/tools/xclbinutil/SectionAIEPartition.cxx

```diff
@@ -103,7 +103,9 @@ SectionAIEPartition::subSectionExists(const std::string& /*sSubSectionName*/) co
 static const std::vector<std::pair<std::string, CDO_Type>> CTTypes = {
   { "UNKNOWN", CT_UNKNOWN },
   { "PRIMARY", CT_PRIMARY },
-  { "LITE", CT_LITE }
+  { "LITE", CT_LITE },
+  { "PRE", CT_PREPOST },
```

Comment:
Why are the PRE and POST associated with the same enum value.   Shouldn't this be "PREPOST" instead?

Being that this is data driven, "PRE" and "POST" entries will be mapped to CT_PREPOST, unfortunately in reverse CT_PREPOST gets mapped to "PRE".

Reply:
Have checked with DPU team, they are good with "PRE_POST" for all pre and post process kernel.

Updated the code